packages/menu-bar/test/overflow.test.js

```diff
@@ -160,22 +143,6 @@ describe('overflow', () => {
       await nextResize(menu);
 
       expect(buttons[0].getAttribute('tabindex')).to.equal('0');
-      expect(buttons[1].getAttribute('tabindex')).to.equal('-1');
```

Comment:
This check is not so relevant for the test, the actual thing is to ensure `tabindex` is set to `0` on the first visible button (which had `tabindex="-1"` before resizing above).